servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/H2PriorKnowledgeFeatureParityTest.java

```diff
@@ -800,7 +799,6 @@ public void serverGracefulClose() throws Exception {
         h1ServerContext.closeAsyncGracefully().subscribe();
 
         try (BlockingHttpClient client2 = forSingleAddress(HostAndPort.of(serverAddress))
-                .socketOption(CONNECT_TIMEOUT, 1) // windows default connect timeout is seconds, we want to fail fast.
```

Comment:
This test looks racy.

Test Thread:
- initiate server graceful close
- client send 2nd request
- assert client 2nd request fails (assumes server graceful close has started, but it hasn't yet)

Server IO thread (or offload thread)
- graceful close state updated (happens after the new connection / request has already started)